### PR TITLE
Don't say that `Recreate` and `Auto` VPA modes are experimental

### LINKED ISSUE #5245: Consider not marking Recreate / Auto VPA modes as experimental
**Which component are you using?**:

VPA

**Is your feature request designed to solve a problem? If so describe the problem this feature should solve.**:
VPA `Recreate` and `Auto` modes are marked as experimental in the [documentation](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler#quick-start). They've been around for a while so we should consider not saying they're experimental anymore.

**Describe the solution you'd like.**:

Check if they're ready & either improve those modes or stop saying they're experimental.

**Describe any alternative solutions you've considered.**:

-

**Additional context.**:

-

### PR DESCRIPTION
**Pull request by jbartosik**
#### Which component this PR applies to?

VPA

#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #5245

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: vertical-pod-autoscaler/README.md [hunk 1]
```diff
@@ -139,15 +139,21 @@ There are four modes in which *VPAs* operate:
   them on existing pods using the preferred update mechanism. Currently, this is
   equivalent to `"Recreate"` (see below). Once restart free ("in-place") update
   of pod requests is available, it may be used as the preferred update mechanism by
-  the `"Auto"` mode. **NOTE:** This feature of VPA is experimental and may cause downtime
-  for your applications.
+  the `"Auto"` mode. **NOTE:** This feature of VPA may cause downtime for your
+  applications if new pods can't start (for example because there is no space for them
+  in the cluster). Using
+  [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#cluster-autoscaler)
+  should mitigagte the problem.
```
Comment (voelzmo) [inline vertical-pod-autoscaler/README.md:146]:
```suggestion
  should mitigate the problem.
```

small typo

File: vertical-pod-autoscaler/README.md [hunk 2]
```diff
@@ -139,15 +139,21 @@ There are four modes in which *VPAs* operate:
...
   them on existing pods by evicting them when the requested resources differ significantly
   from the new recommendation (respecting the Pod Disruption Budget, if defined).
   This mode should be used rarely, only if you need to ensure that the pods are restarted
   whenever the resource request changes. Otherwise, prefer the `"Auto"` mode which may take
   advantage of restart-free updates once they are available. **NOTE:** This feature of VPA
-  is experimental and may cause downtime for your applications.
+  may cause downtime for your applications if new pods can't start (for example because
+  there is no space for the in the cluster). Using
+  [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#cluster-autoscaler)
+  should mitigagte the problem.
```
Comment (voelzmo) [inline vertical-pod-autoscaler/README.md:156]:
```suggestion
  should mitigate the problem.
```

small typo

**Review by voelzmo — CHANGES_REQUESTED** [review]
There's two small typos and I think we also need to change the wording in the **Known Limitations** section as well. It still has 
> Updating running pods is an experimental feature of VPA.

**Comment by jbartosik** [comment]
I can't push fixes to this PR, I'm doing #5293 instead.

**Pull request closed without merging**